packages/n/nushell/package.yml

```diff
@@ -14,9 +14,15 @@ builddeps  :
     - cargo
 setup      : |
     cargo fetch --locked
-build    : |
+build      : |
     cargo build --frozen --release \
                 --workspace \
                 --features=extra,dataframe
 install    : |
     install -Dm00755 ./target/release/nu $installdir/usr/bin/nu
+    install -Dm00755 ./target/release/nu_plugin_custom_values $installdir/usr/bin/nu_plugin_custom_values
```

Comment:
You can use a wildcard here to match all of the plugins, and the `-t` flag of install to install to the folder

So something like:
```
install -Dm00755 ./target/release/nu_plugin_* -t $installdir/usr/bin/
```
Would work. If there are plugins that you don't want to copy then you can do bash shell expansion to pick up only the ones you want. Example:
```
install -Dm00755 ./target/release/nu_plugin_{custom_values,example,formats,gstat,inc,query} -t $installdir/usr/bin/
```

Reply:
Okay adjusted I tried to do a wildcard but likely used the wrong flag or something so went for trusty copy and paste so good to see a working example.

Reply:
I built your latest revision locally and it looks like it's including `.d` files which are not needed from what I can see. They look like some kind of compiler output.